pyproject.toml

```diff
@@ -15,10 +15,10 @@ dependencies = [
     "filetype~=1.2.0",
     "GitPython~=3.1.34",
     "Jinja2~=3.1.2",
-    "Pillow~=10.0.0",
+    "Pillow~=10.0.1",
```

Comment:
`~=` implies use whatever patch version that's available. So it's not really required to bump unless we explicitly want to bump it.

Reply:
It‘s not required, no. But PyPDF will require it, so we should probably do as well, given it‘s just as well a [security fix](https://pillow.readthedocs.io/en/stable/releasenotes/10.0.1.html) and no harm is to be expected.